### ISSUE TITLE
Blacklist language-autotools for Makefiles

### ISSUE THREAD

**Issue by florianb** [sindresorhus/atom-editorconfig#182]
The current version messes the indent_size up (see #181)

**Comment by shellscape** [comment]
Given the age of the issue, I'm not sure this will be resolved. But the package is not respecting rules for `Makefile` in the `.editorconfig` file. Possibly related to this.

**Comment by Alhadis** [comment]
@shellscape Whatever you're observing, it likely isn't related to this issue (@florianb likely forgot to close this after #181 turned out to be [unrelated](https://github.com/sindresorhus/atom-editorconfig/issues/181#issuecomment-281106165).

Please file a new issue for this. Remember to include your local config and list of installed packages.
